src/ch/ch_monitor.c

```diff
@@ -735,6 +736,10 @@ static int virCHMonitorProcessEvent(virCHMonitorPtr mon,
 
     ev = virCHMonitorEventFromString(source, event);
     VIR_DEBUG("Source: %s Event: %s, ev: %d", source, event, ev);
+    virCHDriverPtr driver = CH_DOMAIN_PRIVATE(vm)->driver;
+    /* log start of migration */
```

Comment:
Following link logs all Events from CLH, not just migration. Could you update the comment appropriately?